services/processor/native/test/call_argument_test.go

```diff
@@ -75,7 +75,6 @@ func TestProcessCall_Arguments(t *testing.T) {
 					require.NoError(t, err, "call should succeed")
 					require.Equal(t, tt.expectedOutput, output.OutputArgumentArray, "call return args should be equal")
 				}
-				require.Equal(t, tt.expectedResult, output.CallResult, "call result should be equal")
```

Comment:
why did you remove this line?

Reply:
Because it's not right to assume that if we pass X arguments to the transformer method, it should also return X arguments.